src/controllers/mainController.ts

```diff
@@ -351,7 +351,7 @@ export default class MainController implements vscode.Disposable {
             }
             node = node.parentNode;
         }
-        return Constants.defaultDatabase;
+        return LocalizedConstants.defaultDatabaseLabel;
     }
```

Comment:
And you've verified that this doesn't break new query? It looks like that uses this function to update the connection so be careful to check that it's not going to try and set the DB to <default>.